packet.go

```diff
@@ -256,8 +327,15 @@ func (p *Packet) decodeUdp() {
 	udp.Length = binary.BigEndian.Uint16(pkt[4:6])
 	udp.Checksum = binary.BigEndian.Uint16(pkt[6:8])
 	p.setHeader(udp)
-
 	p.Payload = pkt[8:]
+
+	// potential vxlan packet. vxlan header is 8 bytes.
+	// typical ports are IANA assigned 4789 or linux default 8472.
+	for i := range p.VxLanPorts {
+		if udp.DestPort == p.VxLanPorts[i] && p.decodeVxLAN() {
```

Comment:
We should have the concept of disable (0 value), and disable this for everything other than known CNIs

Reply:
We enforce this in the caller of pcap library. If the slice is empty, we will skip doing any vxlan related work.